config/pipeline.yaml

```diff
@@ -2621,13 +2621,13 @@ scheduler:
     event: *kbuild-gcc-12-arm-node-event
     runtime: *lava-collabora-runtime
     platforms:
-      - bcm2836-rpi-2-b
+      - imx6q-sabrelite
 
   - job: ltp-dio
     event: *kbuild-gcc-12-arm-node-event
     runtime: *lava-collabora-runtime
     platforms:
-      - bcm2836-rpi-2-b
+      - odroid-xu3
```

Comment:
same question here

Reply:
Just putting some other board in place as we want to schedule the job at least on some board.